morf-testsupport/src/main/java/org/alfasoftware/morf/jdbc/AbstractSqlDialectTest.java

```diff
@@ -5064,6 +5065,45 @@ public void testPrepareStatementParameter() throws SQLException {
 
   }
 
+  @Test
+  public void testPortableFunction() {
+    AliasedField function = PortableFunction.withFunctionForDatabaseType(
+            "PGSQL",
+            "TRANSLATE",
+            new FieldReference("field"),
+            new FieldLiteral("1"),
+            new FieldLiteral("A"))
+        .addFunctionForDatabaseType(
+            "H2",
+            "TRANSLATE",
+            new FieldReference("field"),
+            new FieldLiteral("2"),
+            new FieldLiteral("B"))
+        .addFunctionForDatabaseType(
+            "ORACLE",
+            "TRANSLATE",
+            new FieldReference("field"),
+            new FieldLiteral("3"),
+            new FieldLiteral("C"))
+        .addFunctionForDatabaseType(
+            "MY_SQL",
+            "TRANSLATE",
+            new FieldReference("field"),
+            new FieldLiteral("4"),
+            new FieldLiteral("D"))
+        .addFunctionForDatabaseType(
+            "SQL_SERVER",
+            "TRANSLATE",
+            new FieldReference("field"),
+            new FieldLiteral("5"),
+            new FieldLiteral("E"))
+        .as("field");
+
+    UpdateStatement testStatement = UpdateStatement.update(new TableReference("Table")).set(function).build();
+
+    assertEquals(expectedPortableStatement(), testDialect.convertStatementToSQL(testStatement));
+  }

```

Comment:
Function name for every dialect is the same, "TRANSLATE". Therefore, technically,
you are not testing the correct choice of function name by dialect, only the arguments differ.
The implementation keeps function names and arguments inseparable, but you should not count on that.

Reply:
Tweaked test coverate so that this is tested in AbstractSqlDialectTest. I was previously doing this testing in TestPortableSqlFunction, but it makes more sense here.